bundle/tests/scorecard/kuttl/persistent-storage/00-persistent-storage-rc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: persistent-rc
+spec:
+  applicationImage: navidsh/demo-day
+  replicas: 1
+  volumeMounts:
+    - name: pvc
+      mountPath: /data
+  storage:
```

Comment:
Specifying `spec.storage.mountPath: /data` should have similar result. The operator will specify the volume mount automatically. It'll be good to add a test case for that scenario.